client/index.js

```diff
@@ -5,7 +5,7 @@ import {fetchAllProducts} from './reducers';
 import store from './store';
 import Footer from './components/Footer'
 import Navbar from './components/Navbar'
-import Banner from './components/Banner'
+import Banner from './components/B'
```

Comment:
@cylau1031 @knpt this was merged into master and is throwing errors

Reply:
also I think the Banner should only appear on the Home page. Currently it is showing up when we route to `/product/:productId` as well